src/poetry/console/commands/build.py

```diff
@@ -1,23 +1,26 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING
+from pathlib import Path
 
 from cleo.helpers import option
 
 from poetry.console.commands.env_command import EnvCommand
 from poetry.utils.env import build_environment
 
 
-if TYPE_CHECKING:
-    from pathlib import Path
-
-
 class BuildCommand(EnvCommand):
     name = "build"
     description = "Builds a package, as a tarball and a wheel by default."
 
     options = [
-        option("format", "f", "Limit the format to either sdist or wheel.", flag=False)
+        option("format", "f", "Limit the format to either sdist or wheel.", flag=False),
+        option(
+            "output",
+            "o",
+            "Set output directory for build artifacts. Default is `dist`.",
```

Comment:
Normally, we don't use backticks here (only in the docs), but I do not have strong feelings and did not found a reference for something else so 🤷